File: v6/sprefa-extract/tests/fixtures/rust/scip/gamma.rs
//! scip-ratchet fixture: imports alpha's helper through a use, so scip binds
//! the call where the corpus name-match is ambiguous -> ScipOverride (the ts
//! scip/gamma.ts mirror).
pub fn before_reexport() -> u32 {
    0
}

pub use crate::scip::alpha::helper;

pub fn run() -> u32 {
    // A comment can say use without turning this executable reference into an import.
    helper()
}

File: v6/sprefa-extract/tests/fixtures/ts/scip/gamma.ts
// scip-ratchet fixture (4c-ii): calls `helper()` with NO same-file def. The
// name-match sees alpha.helper + beta.helper (ambiguous -> no row); scip
// resolves the call to alpha.helper through the import -> ScipOverride.
import { helper } from "./alpha";

export function use(): number {
  return helper();
}
